imblearn/under_sampling/_prototype_selection/_edited_nearest_neighbours.py

```diff
@@ -174,7 +174,11 @@ def _more_tags(self):
 class RepeatedEditedNearestNeighbours(BaseCleaningSampler):
     """Undersample based on the repeated edited nearest neighbour method.
 
-    This method will repeat several time the ENN algorithm.
+    This method will repeat the ENN algorithm several times. The repetitions
+    will stop when i) the maximum number of iterations is reached, or ii) no
+    more observations are being removed, or iii) one of the majority classes
+    becomes a minority class or iv) one of the majority classes disappears
+    from the target after undersampling.
 
```

Comment:
At the moment, there is no way of understanding how the algo stops, unless we read the source code. This should fix it.